src/components/attendee/AttendeeLayout.module.scss

```diff
@@ -36,7 +36,7 @@ $-z-index--popover: layer.above($-z-index--ui);
 
 .LayerUi {
```

Comment:
[OOS] Just noting this should've been renamed PRs ago into `layerUI` or just `container` or something of the kind